components/header.tsx

```diff
@@ -0,0 +1,31 @@
+import { useWeb3Modal } from "@web3modal/wagmi/react";
+import { Button } from "@aragon/ods";
+import { useAccount, useWalletClient } from "wagmi";
+import { Else, IfCase, Then } from "./if";
+
+const Header = () => {
+  const { isConnected } = useAccount();
+  const { data: client } = useWalletClient();
+  const { open } = useWeb3Modal();
+
+  return (
+    <div className="flex flex-row-reverse w-full mt-5 h-24">
+      <IfCase condition={isConnected}>
+        <Then>
+          <Button suppressHydrationWarning size="md" variant="secondary" onClick={() => open()}>
+            <span suppressHydrationWarning>
+              Connected {client?.chain?.name ? "to " + client?.chain?.name : ""}
```

Comment:
This is not really needed. Let's just put your address here and nothing else.
And ideally, not using a button per se, as it looks a bit odd (even though it is clickable)